routemaster/mixins/redis.rb

```diff
@@ -12,13 +11,10 @@ def self.included(by)
       end
 
       def _redis
-        @@_redis ||= ::Redis::Namespace.new(
-          ENV.fetch('ROUTEMASTER_REDIS_PREFIX'),
-          redis: ::Redis.new(url: ENV['ROUTEMASTER_REDIS_URL']))
-      end
-
-      def _redis_prefix
-        @@_redis_prefix ||= ENV.fetch('ROUTEMASTER_REDIS_PREFIX')
+        @@_redis ||=
+        ConnectionPool.wrap(size: Routemaster.config[:redis_pool_size], timeout: 2) do
```

Comment:
Should we really be using `wrap` here? Is there a plan to migrate to using `.with` rather than wrapping?